packages/components/user/UserCard.tsx

```diff
@@ -1,12 +1,12 @@
 import { Buffer } from "buffer";
```

Comment:
hmm it just make roles a props and remove the fakeRoles,
but indeed, it's more a step that could be done in another PR or in the one for insert role manager.
Should i open another PR ?

Reply:
yes please because the scope of this PR is clearly defined and we can merge the fix right now where the role removal needs a bit more polish when there is no roles IMO